### ISSUE TITLE
iron-component-page as a devDependencies in bower.js

### ISSUE THREAD

**Issue by bartolom** [RenatoUtsch/poly-nvd3#8]
https://github.com/PolymerElements/marked-element/blob/master/bower.json#L29

In https://github.com/RenatoUtsch/poly-nvd3/blob/master/bower.json#L24 there is a reference in the `"dependencies"` section to:
`"iron-component-page": "PolymerElements/iron-component-page#~1.0.5",`
This dependency pulls many other dependcies which are most likely for the nice display of irion-component-page.

```
poly-nvd3#1.4.1 bower_components\poly-nvd3
├── d3#3.5.16
├── d3-import#1.0.0
├── iron-ajax#1.0.9
├── iron-component-page#1.0.9
├── iron-resizable-behavior#1.0.3
├── nvd3#1.8.2
└── polymer#1.4.0

iron-component-page#1.0.9 bower_components\iron-component-p
├── hydrolysis#1.23.1
├── iron-ajax#1.0.9
├── iron-doc-viewer#1.0.14
├── iron-flex-layout#1.3.1
├── iron-icons#1.1.3
├── iron-selector#1.3.0
├── paper-header-panel#1.1.5
├── paper-styles#1.1.4
├── paper-toolbar#1.1.4
└── polymer#1.4.0
```

wouldn't it be suffcient to just add it as a `"devDependencies"` similar to the `marked-element` which is maintained by the PolymerElements team
https://github.com/PolymerElements/marked-element/blob/master/bower.json#L29

```
"iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
```

**Comment by RenatoUtsch** [comment]
I agree, I'll make a commit with the fix.

**Comment by RenatoUtsch** [comment]
Fixed by 4cb0f9e9a. v1.4.2 already has this fix.
